tests/src/Common/Platform/platformdefines.cpp

```diff
@@ -89,7 +89,7 @@ int TP_slen(LPWSTR str)
 	return len;
 }
 
-int TP_scmp_s(LPSTR str1, LPSTR str2)
+int TP_scmp_s(LPCSTR str1, LPCSTR str2)
```

Comment:
Not sure why this function is even needed but if we do use it then it should match the `strcmp` signature.